torch/_inductor/scheduler.py

```diff
@@ -2801,7 +2807,23 @@ def replace_operation_buffer(
                     min_node_unfused,
                     torch._inductor.ir.TritonTemplateCallerBase,
                 ):
-                    node.node.finalize_as_triton_caller(min_node_unfused)
+                    if config.multi_kernel_hints:
+                        callers: dict[Optional[int], TritonTemplateCallerBase] = {}
```

Comment:
How do we decide if say for two of the shapes triton is faster but for another shape cublas is faster ?